File: hard/13_count_inversions/count_inversions_test.go
package main

import "testing"

func TestProgram(test *testing.T) {
	input := []int{2, 3, 3, 1, 9, 5, 6}
	expected := 5
	actual := countInversions(input)
	if actual != expected {
		test.Errorf("Test Failed: input=%v, expected=%v, actual=%v", input, expected, actual)
	}
}

File: hard/13_count_inversions/count_inversions.test.js
const countInversions = require('./count_inversions');

test('Test Case #1', () => {
  const input = [2, 3, 3, 1, 9, 5, 6];
  const expected = 5;
  const actual = countInversions(input);
  expect(actual).toEqual(expected);
});